docs/iris/src/whatsnew/contributions_3.0.0/incompatiblechange_2019-Nov-12_remove_experimental_concatenate_module.txt

```diff
@@ -0,0 +1,3 @@
+* The :mod:`iris.experimental.concatenate` module has now been removed. Previously the experimental `concatenate`
+  functionality was moved to the :meth:`iris.cube.CubeList.concatenate` method as of v1.6.0, and calling the
+  :func:`iris.experimental.concatenate.concatenate` function raised an exception.
```

Comment:
Maybe slightly clearer - but is fine really:
* The :mod:`iris.experimental.concatenate` module has now been removed.  In v1.6.0 the experimental `concatenate`
  functionality was moved to the :meth:`iris.cube.CubeList.concatenate` method.  Calling the:`iris.experimental.concatenate.concatenate` function has raised an exception since then.

Reply:
@abooton I've made a further minor tweak to your excellent revision. Seems reasonable, take a peek...